yb-voyager/cmd/importData_test.go

```diff
@@ -846,6 +853,636 @@ CREATE TABLE public.foo (
+	status, err := dbzm.ReadExportStatus(statusFilePath)
+	if err != nil {
+		utils.ErrExit("Failed to read export status file: %s: %v", statusFilePath, err)
+	}
+	testutils.FatalIfError(t, err, "reading dbzm export status")
+	if status != nil && status.Mode == dbzm.MODE_STREAMING {
+		return true
+	}
+	return false
+}
+
+func snapshotPhaseCompleted(t *testing.T, postgresPass string, targetPass string, snapshotRows int64, tableName string) bool {
+	_, err := testutils.RunVoyagerCommand(nil, "get data-migration-report", []string{
+		"--export-dir", exportDir,
+		"--output-format", "json",
+		"--source-db-password", postgresPass,
+		"--target-db-password", targetPass,
+	}, nil, true)
+	testutils.FatalIfError(t, err, "get data-migration-report command failed")
+
+	reportFilePath := filepath.Join(exportDir, "reports", "data-migration-report.json")
+	// Check if the report file exists and is not empty.
+	if ok := utils.FileOrFolderExists(reportFilePath); ok {
+		jsonFile := jsonfile.NewJsonFile[[]*rowData](reportFilePath)
+		rowData, err := jsonFile.Read()
+		testutils.FatalIfError(t, err, "error reading get data-migration-report")
+		exportSnapshot := 0
+		importSnapshot := 0
+		for _, row := range *rowData {
+			if row.TableName == tableName {
+				if row.DBType == "source" {
+					exportSnapshot = int(row.ExportedSnapshotRows)
+				}
+				if row.DBType == "target" {
+					importSnapshot = int(row.ImportedSnapshotRows)
+				}
+			}
+		}
+		if exportSnapshot == int(snapshotRows) && exportSnapshot == importSnapshot {
+			return true
+		}
+
+	}
+	return false
+}
+
+func streamingPhaseCompleted(t *testing.T, postgresPass string, targetPass string, streamingInserts int64, tableName string) bool {
+	_, err := testutils.RunVoyagerCommand(nil, "get data-migration-report", []string{
+		"--export-dir", exportDir,
+		"--output-format", "json",
+		"--source-db-password", postgresPass,
+		"--target-db-password", targetPass,
+	}, nil, true)
+	testutils.FatalIfError(t, err, "get data-migration-report command failed")
+
+	reportFilePath := filepath.Join(exportDir, "reports", "data-migration-report.json")
+	// Check if the report file exists and is not empty.
+	if ok := utils.FileOrFolderExists(reportFilePath); ok {
+		jsonFile := jsonfile.NewJsonFile[[]*rowData](reportFilePath)
+		rowData, err := jsonFile.Read()
+		testutils.FatalIfError(t, err, "error reading get data-migration-report")
+		exportInserts := 0
+		importInserts := 0
+		for _, row := range *rowData {
+			if row.TableName == `test_schema."test_live"` {
+				if row.DBType == "source" {
+					exportInserts = int(row.ExportedInserts)
+				}
+				if row.DBType == "target" {
+					importInserts = int(row.ImportedInserts)
+				}
+
+			}
+		}
+		if exportInserts == int(streamingInserts) && exportInserts == importInserts {
+			return true
+		}
+
+	}
+	return false
+}
+
+func validateSequence(t *testing.T, startID int64, endId int64, ybConn *sql.DB, tableName string) {
```

Comment:
What does this func do? can you add a comment? 
also, you don't seem to be using startId, endId? 
Are you just checking that these ids all exist on the table?